src/org/mozilla/javascript/NativeError.java

```diff
@@ -234,13 +223,13 @@ private Object callPrepareStack(Function prepare, ScriptStackElement[] stack) {
 
     private static Object js_toString(Scriptable thisObj) {
         Object name = ScriptableObject.getProperty(thisObj, "name");
-        if (name == NOT_FOUND || name == Undefined.instance) {
+        if (name == NOT_FOUND || Undefined.isUndefined(name)) {
```

Comment:
Is name == null possible?

Reply:
null is converted to the string "null" by the ScriptRuntime.toString() call, and that's what the tests expect.